src/main/configSchema.ts

```diff
@@ -116,6 +118,12 @@ export const configSchema = {
     minimum: 15,
     maximum: 60,
   },
+  obsForceMono: {
+    description:
+      'Whether to force the audio of your input device to mono. Enable if your microphone audio is only playing out of  one stereo channel.',
+    type: 'boolean',
+    default: false,
```

Comment:
Should this be true? I can't imagine many people want their mic audio coming through only one speaker. What's the implication if it's already mono and we have obsForceMono enabled?

Reply:
I think one implication would be if the user has a stereo mic and we force it to mono, they would hear the output as mono and thus lose some quality in their recording. Seems like an edge case though and I agree setting it to default to true makes sense